### PR TITLE
Fixes #54 added code of conduct and linked it in readme.md

### LINKED ISSUE #54: Code_Of_Conduct.md not there in the repository
What documentation needs to be updated?
The Code of Conduct documentation for the twinTrim repository.

Describe the changes needed -:

> Add a more detailed Enforcement Guidelines section that clearly outlines how violations of the Code of Conduct will be reported, investigated, and resolved.

> Remove any specific maintainer email in the Reporting Violations section to make it generic for all contributors.

> Ensure that the documentation includes sections on consequences for unacceptable behavior, the right to appeal, confidentiality, and protection from retaliation.

Additional context - :

> The Code of Conduct should follow a recognized standard such as the Contributor Covenant, but be customized for the twinTrim project.

> The goal is to provide clarity on how reports of misconduct will be handled, ensuring contributors understand the process and their rights within the community.
Please assign me this issue so I can add it. Please add gssoc ext and hacktoberfest labels

### PR DESCRIPTION
**Pull request by ShirshenduR**
## Description

This change introduces a **Code of Conduct** to the TwinTrim project, ensuring that all participants in the project adhere to a set of community guidelines promoting respect, inclusivity, and professionalism. It connects the `CODE_OF_CONDUCT.md` file with the project’s `README.md`, informing contributors about its existence and importance.

- Fixes #54 @Kota-Karthik 

## Type of change

- [x] Documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have maintained a clean commit history by using the necessary Git commands
- [x] I have checked that my code does not cause any merge conflicts

## Screenshots (if applicable)

N/A (Not necessary for this documentation change)

### CODE REVIEW

**Comment by techy4shri** [comment]
Rejected due to violations of rules of the program. This is a warning, kindly do not copy paste or else you might get banned from the program.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Code of Conduct
+
+## Our Pledge
```
Comment (techy4shri) [inline CODE_OF_CONDUCT.md:3]: This plege is for a community project repository, not for an individual owned one. Rewrite it or remove it.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+We, as members and maintainers of the **TwinTrim** project, pledge to foster a welcoming, inclusive, and harassment-free environment for everyone involved. Whether you are contributing code, writing documentation, submitting issues, or participating in discussions, we are committed to making the TwinTrim community a positive experience for individuals of all ages, body sizes, visible or invisible disabilities, cultural backgrounds, ethnicities, gender identities and expressions, levels of experience, education, socioeconomic status, nationalities, personal appearances, races, religions, or sexual identities and orientations.
```
Comment (techy4shri) [inline CODE_OF_CONDUCT.md:5]: Again, there is only one maintainer here, and this is extremely lengthy and unnecessary.

**Review by techy4shri — COMMENTED** [review]
This code of conduct is customized for a large project, kindly make changes to suit this current repository.

**Review by techy4shri — CHANGES_REQUESTED** [review]
Unless the changes are made, this cannot be merged.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (techy4shri) [inline CODE_OF_CONDUCT.md:5]: This is for a community, a large one. The repository is individually owned and you need to keep that in mind.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- **Representation**: An individual is considered to be representing the **TwinTrim** project when they are using an official project email, posting via an official social media account, or acting as a spokesperson or representative at any event.
+
+## Enforcement
+
+Instances of inappropriate behavior, harassment, or other violations of this Code of Conduct can be reported by contacting the project team at [your email]. All complaints will be reviewed, investigated, and dealt with promptly, fairly, and confidentially. Steps may include:
+
+1. **Initial Review**: Upon receiving a report, the project maintainers will review the complaint and, if necessary, gather additional information from relevant individuals.
+2. **Resolution**: Based on the investigation, maintainers may choose to issue warnings, request changes in behavior, temporarily suspend individuals from participation, or implement permanent bans for severe or repeated violations.
+3. **Confidentiality**: The project team is committed to protecting the privacy and confidentiality of all parties involved in a complaint, including the reporter.
```
Comment (techy4shri) [inline CODE_OF_CONDUCT.md:54]: this is not a step, either remove it or put the first two in indentation.

**Review by techy4shri — CHANGES_REQUESTED** [review]
Mkae the necessary changes.

**Comment by ShirshenduR** [comment]
@Kota-Karthik Please check

**Review by techy4shri — APPROVED** [review]
Looks good to me.

**Pull request merged**